packages/client/manifest.json

```diff
@@ -4490,6 +4490,16 @@
               "setting": "clickBehaviour",
               "value": "details"
             }
+          },
+          {
+            "type": "boolean",
+            "label": "Do not display default save notification",
```

Comment:
Does this fit in one line? If not then I think we should reword this slightly so that it doesn't wrap.

Reply:
It does not (though neither does the label above it if I remember correctly) I'll try and think of a different wording

Reply:
I had to changed it to 'Hide save alert' to get it fit, not sure if that's acceptable. I thought about inverting the toggle and dropping the verb at the start, to get `Save notification`, but the problem is that also sounds like an action so idk.